grudge/dof_desc.py

```diff
@@ -36,12 +36,14 @@
 .. autoclass:: DTAG_VOLUME_ALL
 .. autoclass:: DTAG_BOUNDARY
 .. autoclass:: QTAG_NONE
+.. autoclass:: QTAG_MODAL
 
 .. autoclass:: DOFDesc
 .. autofunction:: as_dofdesc
 
 .. data:: DD_SCALAR
 .. data:: DD_VOLUME
+.. data:: DD_MODAL
```

Comment:
```suggestion
.. data:: DD_VOLUME_MODAL
```
? I'm also OK with `DD_VOL_MODAL`.